app/lib/audit/models.dart

```diff
@@ -566,23 +566,25 @@ class AuditLogRecord extends db.ExpandoModel<String> {
   }
 
   factory AuditLogRecord.uploaderInvited({
-    required User user,
+    required AuthenticatedAgent agent,
```

Comment:
Can this ever be called by a non-user? Maybe, an admin?
```suggestion
    required AuthenticatedUser agent,
```

Reply:
It can be a `AuthenticatedGcpServiceAccount` (which is the entire reason of the refactor in #6187), as admins are no longer users, and admin-triggered consents should be handled as a non-User accounts.